examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,110 @@
+
+## Prerequisites
+
+* An Amazon S3 bucket as a Terraform backend to store Terraform state data files.
+* An IAM role, or an IAM user principal with required privileges to create resources such as Amazon VPC, Amazon S3 bucket, Amazon EKS cluster, Amazon MSK cluster, and Amazon OpenSearch cluster.
+* The following tools installed on a machine.
+  * Kubectl - for more information, see [Installing or updating kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html).
+  * Terraform - for more information, see [Installing Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli).
+  * Optionally, Apache Kafka client libraries and tools - for more information, see [Create a topic](https://docs.aws.amazon.com/msk/latest/developerguide/create-topic.html).
+
+## Instructions
+
+* Clone the repository
+
+```bash
+git clone https://github.com/aws-samples/amazon-eks-fluent-logging-examples.git
+```
+
+* CD into terraform directory
+
+```bash
+cd amazon-eks-fluent-logging-examples/examples/eks-msk-opensearch/terraform
+```
+
+**Note**. Terraform code will create VPC and all required components.But your OpenSearch dashboard will not be accessible over internet, so you might consider using a AWS client VPN/RDP access to a windows EC2 instance (or any connectivity method to allow you access to dashboard).
+
+* [Optional] Edit 3-variables.tf to create/edit more namespaces and enable logging on them. In this example we are creating two namespaces. "enable_logs_to_es" is a boolean value which when tru will enable logging for the namespace.
+```
+default = [
+    {
+      "name" : "logging",
+      "enable_logs_to_es" = false,
+    },
+    {
+      "name" : "example",
+      "enable_logs_to_es" = true,
+```
+
+
+* Run following terraform commands to create infrastructure. 
+```
+terraform init
+terraform apply
+```
+Terraform apply will ask you for OpenSearch domain master password which you will later use to login to OpenSearch Dashboard.Note it down and keep it safe.
+
+* Deploy a sample nginx pod and service in 'example' namespace.The deployment will help us to generate some logs for samples.
+```
+kubectl config set-context --current --namespace=example
+kubectl apply -f example-deployment.yaml
+kubectl get svc nginx-service-loadbalancer
+```
+**Note down the name of loadBalancer and copy it in your browser and hit it few times to generate access logs.**
+
+* Login to machine which has KAFKA client binary are installed and list KAFKA topics to verify logs_example topic is created.Use following commands to verify your topics and messages in topic.
+ 
+```
+./bin/kafka-topics.sh --bootstrap-server=<<list of your brokers>>  --list
+./bin/kafka-console-consumer.sh --bootstrap-server <<list of your brokers> --topic logs_example    
+```
+Login to your OpenSearch Dashboard as admin and verify the indexes are created for each of namespace enabled to log to OpenSearch. 
```

Comment:
The terraform creates the opensearch cluster? Is there a link we can give here or more instructions on how to do this?